### ISSUE TITLE
Git Clean Option

### ISSUE THREAD

**Issue by ciyer** [SwissDataScienceCenter/renku-jupyterlab-ts#11]
Add a menu option that runs `git clean -i`.

**Comment by rokroskar** [comment]
closed by #8?

**Comment by ciyer** [comment]
No, it is not yet implemented because we need a good way to inform the user what is going to happen and get her approval for something as potentially destructive as clean.
